Index: albumshaper_2.1_src/src/backend/album.h
===================================================================
--- albumshaper_2.1_src/src/backend/album.h
+++ albumshaper_2.1_src/src/backend/album.h	2006-06-01 13:56:48.000000000 +0200
@@ -113,7 +113,7 @@
   int getCreationDay();
 
   ///Updates the creation date to today's date
-  void Album::updateCreationDate();
+  void updateCreationDate();
 
   ///Updates the modification date to today's date
   void updateModificationDate();
